Shared/Samples/Show viewshed from point on map/ShowViewshedFromPointOnMapView.swift

```diff
@@ -0,0 +1,181 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowViewshedFromPointOnMapView: View {
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// The point on the map where the user tapped.
+    @State private var tapLocation: Point?
+    
+    /// The current geoprocessing status.
+    @State private var geoprocessingInProgress = false
+    
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    
+    var body: some View {
+        MapView(map: model.map, graphicsOverlays: [model.inputGraphicsOverlay, model.resultGraphicsOverlay])
+            .onSingleTapGesture { _, tapPoint in
+                tapLocation = tapPoint
+            }
+        // Disables tap while geoprocessing is in progress.
```

Comment:
Just a spacing thing...
```suggestion
            // Disables tap while geoprocessing is in progress.
```

Reply:
Yeah that was bothering me as well, Xcode's code formatter moved it back every time.